lib/dai-plugin-mcd/test/CdpTypeService.spec.js

```diff
@@ -0,0 +1,61 @@
+import Maker from '@makerdao/dai';
+import { mcdMaker } from './helpers';
+const { ETH } = Maker;
+import { REP, DGX } from '../src';
+
+let service;
+
+describe.each([ETH, REP, DGX])('ETH, REP, DGX', collateral => {
+  let cdpType;
+
+  beforeAll(async () => {
+    const maker = await mcdMaker();
+    service = maker.service('cdpType');
+    cdpType = service.getCdpTypeObject(collateral);
+  });
+
+  test(`get total collateral for ${collateral.symbol}`, async () => {
+    const collateral = await cdpType.getTotalCollateral();
+    expect(collateral).toBe(0);
```

Comment:
@tyler17 better late than never: I should have asked you to have these test non-zero values, because as I use them now I'm realizing there's some bug that's causing values to come out with the wrong magnitude. please keep this in mind for the future: check values in your tests that actually exercise the assumptions you've made in the code, in conditions that are reasonably close to expected real-world ones

Reply:
Yeah makes sense, do you think it's worth me updating these tests now?